### DISCUSSION TITLE
DRAKON support and/or concept integration

### DISCUSSION THREAD

**Discussion by ylluminate** [vlang/v#11580] [Ideas]
[DRAKON](http://drakon-editor.sourceforge.net/) ([source](https://github.com/stepan-mitkin/drakon_editor)) is a very useful tool:

> DRAKON is a visual language for specifications from the Russian space program. DRAKON is used for capturing requirements and building software that controls spacecraft.
> 
> The rules of DRAKON are optimized to ensure easy understanding by human beings.
> 
> DRAKON is gaining popularity in other areas beyond software, such as medical textbooks. The purpose of DRAKON is to represent any knowledge that explains how to accomplish a goal.
> 

It would be great to either add `V` to DRAKON or to add DRAKON functionality to a `V` IDE (or both!).

**Comment by ArnCarveris** [comment]
Very close to `V`: https://github.com/phr34k/vdsl
